src/Evolution/Systems/Cce/AnalyticSolutions/SphericalMetricData.hpp

```diff
@@ -47,6 +47,22 @@ struct SphericalMetricData : public WorldtubeData {
   explicit SphericalMetricData(const double extraction_radius) noexcept
       : WorldtubeData{extraction_radius} {}
 
+  /*!
+   * Computes and returns by pointer the Jacobian
+   * \f$\partial x_{\mathrm{Cartesian}}^j /
+   * \partial x_{\mathrm{spherical}}^i\f$
```

Comment:
I think it would be useful to add the analytic expression for the Jacobians here in the docs.

It's not really necessary to prefix the docs with "Computes and returns by pointer" since returning by reference is pretty standard